modules/stanford_earth_capx/src/Plugin/migrate/process/StanfordEarthCapProfileImage.php

```diff
@@ -0,0 +1,132 @@
+<?php
```

Comment:
My linter has picked up a few errors and warnings on this file:
```
Error	PHPCS	[Drupal.Commenting.DocComment.ShortFullStop] Doc comment short description must end with a full stop	19:4
Error	PHPCS	[Drupal.Commenting.DocComment.LongFullStop] Doc comment long description must end with a full stop	26:4
Error	PHPCS	[Drupal.Commenting.DocComment.SpacingAfterTagGroup] There must be a single blank line after a tag group	42:4
Error	PHPCS	[Drupal.Commenting.DocComment.TagsNotGrouped] Tags must be grouped together in a doc comment	74:4
Error	PHPCS	[Drupal.Commenting.InlineComment.NotCapital] Inline comments must start with a capital letter	102:5
Error	PHPCS	[Drupal.Commenting.InlineComment.InvalidEndChar] Inline comments must end in full-stops, exclamation marks, colons, question marks, or closing parentheses	102:5
Error	PHPCS	[Drupal.ControlStructures.ControlSignature.NewlineAfterCloseBrace] Expected newline after closing brace	107:5
Warning	PHPCS	[Drupal.Classes.UnusedUseStatement.UnusedUse] Unused use statement	7:1
Warning	PHPCS	[Drupal.Classes.UnusedUseStatement.UnusedUse] Unused use statement	10:1
Warning	PHPCS	[Drupal.Classes.UnusedUseStatement.UnusedUse] Unused use statement	14:1
Warning	PHPCS	[Drupal.Classes.UnusedUseStatement.UnusedUse] Unused use statement	15:1
```

Reply:
Fixed.

Reply:
Thank you.